app/src/components/superheroes/SuperheroList.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <ul v-if="superheroes">
     <li v-for="superhero in superheroes" :key="superhero.name">
```

Comment:
```suggestion
    <li v-for="superhero in superheroes" :key="superhero.id">
      <RouterLink :to="`/superheroes/${superhero.id}`">{{superhero.name}}</RouterLink>
```
Two things here: 1. Should always bind a unique key to a loop, ie id, 2. Path to detail should also be unique, ie what if there are two Batman's.